<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QDialog" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== Fondo general ===== */
QDialog#AuthWindow {
  background: qlineargradient(x1:0,y1:0,x2:1,y2:1, stop:0 #0b1210, stop:1 #0f1c17);
  color: #E8EEF7;
  font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, Arial;
}

/* ===== Tarjeta ===== */
QFrame#cardLogin {
  background: #ffffff;
  border: 1px solid #d1d5db;
  border-radius: 14px;
  padding: 24px;
  max-width: 420px;
}

/* ===== Título ===== */
QLabel#lblTitleLogin {
  color: #0b3d2e;
  font-size: 30px;
  font-weight: 800;
  letter-spacing: .5px;
  border: none;
}

/* ===== Labels ===== */
QLabel#lblUser, QLabel#lblPwd {
  color: #334155;
  font-weight: 600;
}

/* ===== Campos ===== */
QLineEdit#txtUsuario, QLineEdit#txtPassword {
  background: #f8fafc;
  color: #111827;
  border: 1px solid #cbd5e1;
  border-radius: 10px;
  padding: 10px 12px;
}
QLineEdit#txtUsuario:focus, QLineEdit#txtPassword:focus {
  border: 2px solid #22c55e;
  background: #ffffff;
}
QLineEdit#txtUsuario::placeholder, QLineEdit#txtPassword::placeholder {
  color: #94a3b8;
}

/* ===== Botón principal (Login) ===== */
QPushButton#btnLogin {
  background: #16a34a;
  color: #ffffff;
  border: 1px solid #15803d;
  border-radius: 10px;
  padding: 10px 16px;
  font-weight: 600;
}
QPushButton#btnLogin:hover   { background: #17b455; }
QPushButton#btnLogin:pressed { background: #12803e; }
QPushButton#btnLogin:disabled{
  background: #a7f3d0; color: #065f46; border-color: #86efac;
}

/* ===== Botón secundario (Registro) ===== */
QPushButton#btnRegister {
  background: transparent;
  color: #0b3d2e;
  border: 1px solid #94a3b8;
  border-radius: 10px;
  padding: 10px 16px;
  font-weight: 600;
}
QPushButton#btnRegister:hover {
  border-color: #16a34a;
  color: #065f46;
}
/* ===== Tarjeta ===== */
QFrame#cardRegister {
  background: #ffffff;
  border: 1px solid #d1d5db;
  border-radius: 14px;
  padding: 24px;
  max-width: 420px;
}

/* ===== Título ===== */
QLabel#lblTitleRegister {
  color: #0b3d2e;
  font-size: 30px;
  font-weight: 800;
  letter-spacing: .5px;
  border: none;
}

/* ===== Labels ===== */
QLabel#lblPwd1, QLabel#lblPwd2, QLabel#lblUserR {
  color: #334155;
  font-weight: 600;
}

/* ===== Campos ===== */
QLineEdit#txtUserRegister, QLineEdit#txtPwdRegister, QLineEdit#txtPwdValidate {
  background: #f8fafc;
  color: #111827;
  border: 1px solid #cbd5e1;
  border-radius: 10px;
  padding: 10px 12px;
}
QLineEdit#txtUserRegister:focus, QLineEdit#txtPwdRegister:focus, QLineEdit#txtPwdValidate:focus {
  border: 2px solid #22c55e;
  background: #ffffff;
}
QLineEdit#txtUserRegister::placeholder, QLineEdit#txtPwdRegister::placeholder,QLineEdit#txtPwdValidate::placeholder {
  color: #94a3b8;
}

/* ===== Botón principal (Register) ===== */
QPushButton#btnRegisterR {
  background: #16a34a;
  color: #ffffff;
  border: 1px solid #15803d;
  border-radius: 10px;
  padding: 10px 16px;
  font-weight: 600;
}
QPushButton#btnLoginR:hover   { background: #17b455; }
QPushButton#btnLoginR:pressed { background: #12803e; }
QPushButton#btnLoginR:disabled{
  background: #a7f3d0; color: #065f46; border-color: #86efac;
}

/* ===== Botón secundario (Login) ===== */
QPushButton#btnLoginR {
  background: transparent;
  color: #0b3d2e;
  border: 1px solid #94a3b8;
  border-radius: 10px;
  padding: 10px 16px;
  font-weight: 600;
}
QPushButton#btnRegisterR:hover {
  border-color: #16a34a;
  color: #065f46;
}

</string>
  </property>
  <widget class="QStackedWidget" name="stackAuth">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>100</y>
     <width>361</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>400</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageLogin">
    <widget class="QFrame" name="cardLogin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>400</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>470</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="1,2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <property name="spacing">
       <number>12</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblTitleLogin">
        <property name="font">
         <font>
          <family>Bebas Neue</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>INICIAR SESION</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblUser">
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtUsuario">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPwd">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Contraseña</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtPassword">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnRegister">
        <property name="text">
         <string>Registrarse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnLogin">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Iniciar Sesion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageRegister">
    <widget class="QFrame" name="cardRegister">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>400</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>470</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0" columnstretch="2,3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <property name="spacing">
       <number>12</number>
      </property>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnRegisterR">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Registrarse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtPwdRegister">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPwd1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Contraseña</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtUserRegister">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnLoginR">
        <property name="text">
         <string>Iniciar Sesion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblUserR">
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblTitleRegister">
        <property name="font">
         <font>
          <family>Bebas Neue</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>REGISTRO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblPwd2">
        <property name="text">
         <string>Repetir Contraseña</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPwdValidate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
